src/ElementQueries.js

```diff
@@ -194,9 +194,12 @@
 
         /**
          * Start the magic. Go through all collected rules (readRules()) and attach the resize-listener.
+         *
+         * @param {HTMLElement} container only elements of the container are considered (document.body if not set)
+         *
          */
-        function findElementQueriesElements() {
-            var query = getQuery();
+        function findElementQueriesElements(container) {
+            var query = (container) ? container.querySelectorAll.bind(container) : getQuery();
```

Comment:
As far as I understood jQuery or Mootools are only used if document.activeElement is not available. According to https://developer.mozilla.org/en-US/docs/Web/API/Document/activeElement it is available in IE4+, Chrome 2+, Firefox 3.0+ etc.
That's the reason for skipping this if a container is available (I also wrote #157 for removing the legacy code)

Should I still add it to `getQuery()`?